### ISSUE TITLE
Error handing for incorrectly-specified conditions/replicates

### ISSUE THREAD

**Issue by AnnaSyme** [Victorian-Bioinformatics-Consortium/degust#48]
On the configuration page, when specifying conditions and their replicates: 

If only two conditions are specified (with one replicate in each), or each replicate is incorrectly saved as a "condition", the error is:

Error in approxfun(l, rule = 2) : 
need at least two non-NA values to interpolate
Calls: voom -> approxfun
Execution halted

@tseemann suggests enhancement to catch this error better by telling user that each condition needs several replicates? (I know it does already say this in the uploading instructions though!).

**Comment by drpowell** [comment]
Yes, better error reporting there is a good idea.  You don't actually need replicates for every condition (although you should)
